src/main/java/ro/itschool/ema/exceptions/ExceptionHandlerAdvice.java

```diff
@@ -0,0 +1,47 @@
+package ro.itschool.ema.exceptions;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.Map;
+
+@Slf4j
+@ControllerAdvice
+public class ExceptionHandlerAdvice {
+
```

Comment:
the exception handlers from this class need to be moved to the GlobalExceptionHandler and modified accordingly. If it is not clear how, merge to main as it is and I will do it afterewards.

Reply:
Solved.